root/static/scripts/common/utility/cloneDeep.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * @flow
```

Comment:
It'd be impossible, currently, I think. (I did make an attempt.) In fact a `clone` function is [the example they use](https://flow.org/en/docs/types/casting/#type-casting-through-any-) when demonstrating a use of `any`:
> It would be hard to create a type for this because we’re creating a new object based on another object.